site/content/en/docs/components/_index.md

```diff
@@ -5,69 +5,196 @@ weight: 30
 
 ## Prow Images
 
-This directory includes a sub directory for every Prow component and is where all binary and container images are built. You can find the `main` packages here. For details about building the binaries and images see ["Building, Testing, and Updating Prow"](/docs/build-test-update/).
+This directory includes a sub directory for every Prow component 
```

Comment:
I'm not a fan of linebreaks in markdown, they make diffs messy. I'd prefer to not do this change.